frame/support/procedural/src/pallet/expand/storage.rs

```diff
@@ -210,6 +278,18 @@ pub fn expand_storages(def: &mut Def) -> proc_macro2::TokenStream {
 					}
 				)
 			},
+			Metadata::CountedMap { key, value } => {
```

Comment:
I was expecting this to expand to the metadata of two separate storage items, one the map and the one the counter.

Reply:
found it.